recipe/meta.yaml

```diff
@@ -10,34 +10,40 @@ source:
   sha256: 887b6433a30282cb002117da89b05812c770fd9469f93950ff3866ddd02bfc64
 
 build:
-  number: 0
+  number: 1
+  # Anaconda's s390x does not contain depdendencies for libgpuarray 0.7.6, this package is skipped for s390x at this time.
```

Comment:
Not important, but I suggest phrasing this "libgpuarray 0.7.6 cannot be built on s390x because of other missing dependencies...".